README.md

```diff
@@ -1,26 +1,31 @@
-# Background
-Under most circumstances, manually downloading this repository should never
-be required.
+# Deprecation note
 
-# Prerequisites
-# Base
-* [Google Protocol Buffers](https://developers.google.com/protocol-buffers)
+This repository used to contain the [protocol
+buffer](https://developers.google.com/protocol-buffers) code that defined both
+the data model and the exposition format of Prometheus metrics.
 
-## Java
-* [Apache Maven](http://maven.apache.org)
-* [Prometheus Maven Repository](https://github.com/prometheus/io.prometheus-maven-repository) checked out into ../io.prometheus-maven-repository
+Starting with v2.0.0, the [Prometheus
+server](https://github.com/prometheus/prometheus) does not ingest the
+protobuf-based exposition format anymore. Currently, all but one of the
+[official instrumentation
+libraries](https://prometheus.io/docs/instrumenting/clientlibs/) do not expose
+the protobuf-based exposition format. The [Go instrumentation
+library](https://github.com/prometheus/client_golang), however, has been built
+around the protobuf-based data model. As a byproduct thereof, it is still able
+to expose the protobuf-based exposition format. The Go instrumentation library
+is the only remaining Prometheus project directly using this repository.
```

Comment:
official

There's a few things outside this org using it I gather.

Reply:
I thought "Prometheus project" reads as "project in the Prometheus GitHub org". I'm not a great fan of the word "official". I'll reword as "…is the only remaining project within the Prometheus GitHub org…".